### PR TITLE
add a msg when a test timeouts

### LINKED ISSUE #1260: ramen test should warn when a test failed by a timeout

### PR DESCRIPTION
**Pull request by darlentar**
closes #1260

### CODE REVIEW

**Comment by rixed** [comment]
I don't get the idea. Each and every time `ramen test` complain that it cannot find a tuple that was required for the test to pass, then a time out occurred.
The timeout is not the error. We have to use a timeout otherwise we would wait for the expected tuple for ever and ever, but the error is that the expected tuple have not been met, not that we decided to save the user some time and cut short the wait.
In other words, the timeout is an implementation detail, an optimisation, and there is no reason to draw user's attention on it.
I think you have been confused by the name "timeout" into thinking that it was an actual error.
